plugins/Goals/tests/System/expected/test_trackGoals_pages__Actions.getEntryPageUrls_day.xml

```diff
@@ -17,11 +17,11 @@
 			<row idgoal='1'>
 				<nb_conversions>5</nb_conversions>
 				<revenue>50</revenue>
-				<nb_conv_pages_before>16</nb_conv_pages_before>
+				<nb_conv_pages_before>11</nb_conv_pages_before>
```

Comment:
is it expected that these numbers change?

Reply:
The fixture was updated to have all the multi-day conversions fall within a calendar week so that other tests could be improved to check week level aggregation and not just a single day.